components/eam/src/physics/crm/sam/crm_module.F90

```diff
@@ -1231,6 +1218,10 @@ subroutine crm(lchnk, ncrms, dt_gl, plev,       &
       crm_output%qltend (icrm,k) =      (qln  (icrm,k) - crm_input%ql  (icrm,k)) * icrm_run_time
       crm_output%qcltend(icrm,k) =      (qccln(icrm,k) - crm_input%qccl(icrm,k)) * icrm_run_time
       crm_output%qiltend(icrm,k) =      (qiiln(icrm,k) - crm_input%qiil(icrm,k)) * icrm_run_time
+#if defined(MMF_MOMENTUM_FEEDBACK)
```

Comment:
Is the vertical index k missing in crm_output%ultend and crm_output%vltend ?

Reply:
good catch, I'll fix it.